src/SDKs/CognitiveServices/dataPlane/Knowledge/QnAMaker/Generated/Alterations.cs

```diff
@@ -0,0 +1,339 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
```

Comment:
The directory structure seems inconsistent.
This is how it should look.
Knowledge -> QnAMaker
Knowledge -> QnAMaker -> Generated
Knowledge -> QnAMaker -> Customization
Knowledge -> QnAMaker -> Properties
Knowledge -> QnAMaker.Tests
Knowledge -> QnAMaker.Tests->SessionRecords

Reply:
done as discussed